apps/site/assets/js/timetable-stick.js

```diff
@@ -0,0 +1,124 @@
+/**
+ * When scrolling down the timetable,
+ * ensure the train numbers are always visible on screen
+ */
+
+const trainLabels = Array.from(
+  document.querySelector(".js-timetable-train-labels").children
+);
+const tableEl = document.getElementById("timetable");
+const tableBodyEl = document.querySelector(".m-timetable tbody");
+const stickyHeaderEl = document.createElement("div");
+stickyHeaderEl.setAttribute(
+  "style",
+  "border: 1px solid #b0b5c0; overflow: hidden; position: fixed; top: 0;"
+);
```

Comment:
Could we use `stickyHeaderEl.className = [class name]` to "extract" this styling or it doesn't really make sense here?

Reply:
Ha, that was my initial approach!  I ultimately decided that since I don't anticipate using this styling anywhere else, I may as well inline it and leave our spreadsheets alone. 🤷‍♀️

Reply:
Ok sounds good.